onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1000,14 +1000,14 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
         .SetDoc(GroupQueryAttention_ver1_doc)
         .Attr("num_heads", "Number of attention heads for q", AttributeProto::INT)
         .Attr("kv_num_heads", "Number of attention heads for k and v", AttributeProto::INT)
-        .Attr("unidirectional",
-              "Whether every token can only attend to previous tokens. Default value is 1.",
-              AttributeProto::INT,
-              static_cast<int64_t>(1))
         .Attr("is_past_bsnh",
               "Whether past kv uses BSNH, otherwise BNSH. Default value is 1 (BSNH).",
               AttributeProto::INT,
               static_cast<int64_t>(1))
+        .Attr("kv_share_buffer",
+              "Whether past kv shares tensor with present kv.",
+              AttributeProto::INT,
+              static_cast<int64_t>(1))
```

Comment:
why do we need this? User needs to generate different models. we can get this just from if past_kv and present_kv have same address.